install/kubernetes/tetragon/templates/_container_tetragon.tpl

```diff
@@ -51,6 +51,7 @@
       mountPath: {{ .mountPath }}
       readOnly: {{ .readOnly }}
 {{- end }}
+    {{- include "tetragon.volumemounts.extra" . | nindent 4 }}
```

Comment:
Does it make sense to pass the output of `include` to `toYaml` before indenting? eg:
```
 {{- include "tetragon.volumemounts.extra" . | toYaml | nindent 4 }}
 ``

Reply:
I would expect these extensions to be yaml already, but I guess it won't hurt

Reply:
No, `toYaml` converts templates to string literals actually. I think it's less error-prone to define yaml correctly in the extension and avoid complicating it here.